### PR TITLE
[13.0][MIG] account_invoice_constraint_chronology

### PR DESCRIPTION
**Pull request by alexey-pelykh**
This migration involved rewrite of the module since v13 breaking changes to invoicing, thus addition to the authors.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
@ThomasBinsfeld @sbidoul if possible, please review this one

**Review by rp-landoo — APPROVED** [review]

File: account_invoice_constraint_chronology/model/account_journal.py [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-    check_chronology = fields.Boolean(default=False,)
+    check_chronology = fields.Boolean()
 
     @api.onchange("type")
     def _onchange_type(self):
-        self.ensure_one()
-        if self.type not in ["sale", "purchase"]:
-            self.check_chronology = False
+        super()._onchange_type()
+        self.check_chronology = self.type in ("sale", "purchase")
```
Comment (sbidoul) [inline account_invoice_constraint_chronology/model/account_journal.py]: This changes the logic a little bit. No big deal I guess. Any particular reason to change ?

**Comment by ghost** [comment]
Hello @pedrobaeza. Can you merge this? We have checked it and we think it runs properly. Thanks a lot.
Regards,

**Review by sbidoul — APPROVED** [review]
LGTM (high level code review, no test)

File: account_invoice_constraint_chronology/model/account_journal.py (hunk 1 above)
Comment (alexey-pelykh) [reply account_invoice_constraint_chronology/model/account_journal.py]: Valid point, there's no big value in changing that, rolled those changes back

**Comment by alexey-pelykh** [comment]
@sbidoul Why patch and not nobump? Or did I miss something?

**Comment by sbidoul** [comment]
Ah yes, I tend to always do patch so the last commit has the version bump and we get a round number in the published wheel.

**Pull request merged**